src/e2e-test/resources/pluginParameters.properties

```diff
@@ -114,3 +118,13 @@ account.schema= [{"key":"Id","value":"string"},{"key":"IsDeleted","value":"boole
   {"key":"JigsawCompanyId","value":"string"},{"key":"AccountSource","value":"string"},\
   {"key":"SicDesc","value":"string"}]
 
+CreateBQTableQueryFile=BigQuery/BigQueryCreateTableQuery.txt
+InsertBQDataQueryFile=BigQuery/BigQueryInsertDataQuery.txt
+
+bigQueryDatatypesColumnsList=(Id,Name,Col_Timestamp__c,Col_Date__c,Col_Currency__c,Col_Email__c,Col_Number__c,\
+  Col_GeoLocation__Latitude__s,Col_GeoLocation__Longitude__s,Col__c,Col_Url__c,Col_Time__c,Col_Text__c)
```

Comment:
I see you are creating the data here as well as in the text files. I will suggest use a single method for data creation.